tests/build/chains.go

```diff
@@ -153,26 +152,19 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", Label("ec"), fu
 				publicKey, err := kubeController.GetPublicKey("signing-secrets", "tekton-chains")
 				Expect(err).ToNot(HaveOccurred())
 				GinkgoWriter.Println("Copy public key from tekton-chains/signing-secrets to a new secret")
-				Expect(kubeController.CreateOrUpdateSigningSecret(
-					publicKey, publicSecretName, namespace)).To(Succeed())
+				Expect(kubeController.CreateOrUpdateSigningSecret(publicKey, publicSecretName, namespace)).To(Succeed())
 
 				rekorHost, err = kubeController.GetRekorHost()
 				Expect(err).ToNot(HaveOccurred())
 				GinkgoWriter.Printf("Configured Rekor host: %s\n", rekorHost)
+
+				// make sure that the buildah pipeline with the produced the
+				// required images
+				err = kubeController.AwaitAttestationAndSignature(imageWithDigest, attestationTimeout)
```

Comment:
Should we remove the test `creates signature and attestation` from above?

Btw, the tests are [configured](https://github.com/redhat-appstudio/e2e-tests/blob/c0cc21839772cd8ca4df3d5e569a1dd5d8e854fb/pkg/framework/describe.go#L23) to be [Ordered](https://onsi.github.io/ginkgo/#ordered-containers). So either approach is fine, but we should probably stick to one.

Reply:
Naah, I'll remove it from here, I think I was seeing something and addressing it with this, will test though for flakiness...